.obs/chartfile/operator/values.yaml

```diff
@@ -10,8 +10,9 @@ seedImage:
   imagePullPolicy: IfNotPresent
 
 channel:
-  image: "%%IMG_REPO%%/rancher/elemental-channel"
-  tag: "%VERSION%"
+  name: "sle-micro-5.5-channel"
```

Comment:
Adding a name here so this turns to be customizable too. I like seeing the OS stream in the channel name, it makes everything way more obvious in CLI and the UI.

Reply:
Really nice